Cargo.lock

```diff
@@ -11,6 +11,15 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "ansi_term"
```

Comment:
Øhh nu når jeg ser det her, er det så virkelig meningen at vi smider en automatisk genereret fil som cargo.lock op på github?

Reply:
Det korte svar er ja. Det lange svar kan du finde:
- https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries
- https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html